web3.js/src/transaction.ts

```diff
@@ -16,6 +15,14 @@ import {toBuffer} from './util/to-buffer';
  */
 export type TransactionSignature = string;
 
+/**
+ * Transaction signer
+ */
+export interface TransactionSigner {
```

Comment:
Nit: `Signer` would align with the Rust SDK

Reply:
Done!